docs/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```

Comment:
I'm starting to think that we should omit changelog changes from the individual PRs so that they wouldn't conflict and just do changelog once before release.

Reply:
As you want, I think the conflict resolution is pretty simple in `docs/changelog.md`.